docs.openc3.com/docs/getting-started/key-concepts.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_position: 5
+sidebar_position: 1
```

Comment:
Moved this to #1 because key-concepts is important to read before installation. It's just the order in the drop down ... no links have changed.